opus/application/static_media/js/widgets.js

```diff
@@ -738,6 +821,14 @@ var o_widgets = {
                        $(mult_id).next().slideDown("fast");
                    }
                });
+
```

Comment:
Just for future reference, these few lines above are also where we should fix issue #663 that's assigned to you.